frontend/src/components/elements/DataFrame/DataFrameUtil.tsx

```diff
@@ -288,6 +304,7 @@ export function getWidths(
   }
 
   const elementWidth = Math.min(distributedTableTotal, containerWidth)
+  const needsHorizontalScrollbar = distributedTableTotal > containerWidth
```

Comment:
Not related at all to your change, but wow getWidths() is a crazy complicated function, it makes my head hurt just looking at it. Wonder if there's some opportunity for cleanup down the line.